student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [OPTIONAL] You want to consider writing the parameters from Person to Student in the initialize and super(). 
`age, classroom, name, parent_permission = true` 🔨

Reply:
In the documentation I read about using super, you only list if you want to use some of the initialized parameters from the parent class. In this case, we are using all parameters, so using `super()` is the best. 👍